src/lib/entities/audio/specs/audio-instance.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { get } from 'svelte/store';
+import { src, currentTime, paused, move, pause, play, seek } from '..';
+
+describe('Audio playback API', () => {
+  const defaultSrc = 'http://commondatastorage.googleapis.com/codeskulptor-assets/Evillaugh.ogg';
+  const playDefault = () => play(defaultSrc);
+
+  it('load audio', () => {
```

Comment:
nitpick (non-blocking): the code would read slightly better like this:
```suggestion
  it('loads audio', () => {
```

Reply:
I meant to do this for all `it()`-s :)